admin/schema_changes/9.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE review ADD COLUMN published_on TIMESTAMP;
```

Comment:
We'd also want to fill up the `published_on` column with values for old reviews. 

And this should be inside a transaction.